src/client/firefox/commands.js

```diff
@@ -407,7 +474,72 @@ async function checkServerSupportsListWorkers() {
   return Services.vc.compare(version, "61.0") >= 0;
 }
 
+async function updateWorkerClients() {
+  const newWorkerClients = {};
+
+  // Temporary workaround for Bug 1443550
+  // XXX: Remove when FF60 for Android is no longer used or available.
+  const supportsListWorkers = await checkServerSupportsListWorkers();
+
+  // NOTE: The Worker and Browser Content toolboxes do not have a parent
+  // with a listWorkers function
+  // TODO: there is a listWorkers property, but it is not a function on the
+  // parent. Investigate what it is
+  if (
+    !threadClient._parent ||
+    typeof threadClient._parent.listWorkers != "function" ||
+    !supportsListWorkers
+  ) {
+    workerClients = newWorkerClients;
+    return;
+  }
+
+  const { workers } = await threadClient._parent.listWorkers();
+  for (const workerTargetFront of workers) {
+    await workerTargetFront.attach();
+    const [, workerThread] = await workerTargetFront.attachThread();
+
+    if (workerClients[workerThread.actor]) {
+      if (workerClients[workerThread.actor].thread != workerThread) {
+        throw new Error(`Multiple clients for actor ID: ${workerThread.actor}`);
+      }
+      newWorkerClients[workerThread.actor] = workerClients[workerThread.actor];
+    } else {
+      addThreadEventListeners(workerThread);
+      workerThread.url = workerTargetFront.url;
+      workerThread.resume();
+
+      const [, consoleClient] = await debuggerClient.attachConsole(
+        workerTargetFront.targetForm.consoleActor,
+        []
+      );
+
+      newWorkerClients[workerThread.actor] = {
+        url: workerTargetFront.url,
+        thread: workerThread,
+        console: consoleClient
+      };
+    }
+  }
+
+  workerClients = newWorkerClients;
+}
```

Comment:
it would be nice to move this to its own util because it will likely be extracted later anyways ...